zenml/cli/config.py

```diff
@@ -56,15 +55,12 @@ def opt_out(config):
 
 
 @config.command("list")
-def list_config():
+@click.pass_context
+def list_config(ctx):
```

Comment:
This function raised a question in my head regarding the way we talk about the local and global config. Tools such as git use the same terminology but in their case the local and global configs are interchangeable. People have the ability to overwrite the global config with the local config per-repository. But in our case, they represent different things, as a result, this method will only display so-called local config. If we change the naming, it might be a bit easier to understand.

Reply:
Hmm fair point. We can still make the global config identical to the local config (except for some flags) in the future. What names would you give it for now? I personally would stick to whats there until someone complains